### PR TITLE
[feature](cloud) export fdb client busyness

### PR DESCRIPTION
**Pull request by w41ter**
a new bvar `fdb_client_thread_busyness_percent` (integer between 0 and 100) will be present when 
```
curl ms_ip:ms_port/vars
```

### CODE REVIEW

**Comment by w41ter** [comment]
run buildall

File: cloud/src/common/bvars.cpp [hunk 1]
```diff
@@ -177,6 +177,7 @@ bvar::Status<int64_t> g_bvar_fdb_workload_transactions_committed_hz(
         "fdb_workload_transactions_committed_hz", BVAR_FDB_INVALID_VALUE);
 bvar::Status<int64_t> g_bvar_fdb_workload_transactions_rejected_hz(
         "fdb_workload_transactions_rejected_hz", BVAR_FDB_INVALID_VALUE);
+bvar::Status<double> g_bvar_fdb_client_thread_busyness("fdb_client_thread_busyness", 0.0);
```
Comment (gavinchou) [inline cloud/src/common/bvars.cpp:180]: make it an integer, consider naming something related to 'percent'

**Comment by w41ter** [comment]
run buildall

File: cloud/src/common/metric.cpp [hunk 2]
```diff
@@ -228,8 +229,14 @@ static void export_fdb_status_details(const std::string& status_str) {
 }
 
 void FdbMetricExporter::export_fdb_metrics(TxnKv* txn_kv) {
+    int busyness = 0.0;
```
Comment (gavinchou) [inline cloud/src/common/metric.cpp:232]: `int` and `0.0` typo?

**Comment by w41ter** [comment]
run buildall

File: cloud/src/common/metric.cpp [hunk 3]
```diff
@@ -228,8 +229,14 @@ static void export_fdb_status_details(const std::string& status_str) {
 }
 
 void FdbMetricExporter::export_fdb_metrics(TxnKv* txn_kv) {
+    int busyness = 0;
```
Comment (gavinchou) [inline cloud/src/common/metric.cpp:232]:
```suggestion
    int64_t busyness = 0;
```

**Comment by gavinchou** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by bobhan1 — APPROVED** [review]

**Pull request merged**